src/Formatter/FormatterInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace Middlewares\Formatter;
+
+use Throwable;
+
+interface FormatterInterface
+{
+    /**
+     * Get the output content type
+     */
+    public function contentType(): string;
```

Comment:
this should return an array of strings, in order to be used for various content types. For example, json error handler could be used by `application/json`, `application/manifest+json`, `application/vnd.geo+json` and so on.

Reply:
This is for the output content type. I can add a separate `acceptTypes(): array` method to handle input. **However** accept type is often more complicated than just a simple list of matches. Maybe it should be a regex instead? Eg:

```php
public function acceptTypes(): string
{
    return 'application/*json*';
}
```